ztp/source-policy-crs/PaoSubscriptionFull.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: v1
```

Comment:
This would either be 
1) The redhat operators catalog or 
2) The catalog the user sets up when using a disconnected registry